content/intro/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: आरंभ करना
+description: Brigade के साथ काम कैसे शुरू करें
+section: परिचय
+aliases:
+  - /index.md
+  - /intro/index.md
+  - /intro/index/
+---
+
+Brigade में नए हैं या जल्दी से गति प्राप्त करना चाह रहे है? आप सही जगह पर आए हैं| जल्दी से काम शुरू करने के लिए यह पढ़े |
+
+## कैसे प्रलेखन सुन्योजित किआ गया है
+
+सभी की जरूरतों को पूरा करने के लिए Brigade की मुक्य प्रलेखन को कुछ टुकड़ों में तोड़ा गया है:
+
+यह परिचयात्मक सामग्री जो दी गयी है, वह उन लोगो के लिए है जो Brigade में नैय है. इसमें कुछ भी गहराई में नहीं बताया गया है, बल्कि इस्कि एक उच्च स्तरीय अवलोकन की गई है की Brigade को कैसे तैनात और इंटरैक्ट करना है, कैसे प्रोजेक्ट बनाना है और घटनाओं को संभालने वाली स्क्रिप्ट कैसे लिखनी है|
+
+दूसरी ओर [विषय गाइड], Brigade के अलग-अलग हिस्सों में गहराई तक जाता है | Brigade की कम्पलीट गाइड, कोर कॉम्पोनेन्ट, सिस्टम रोल्स एट्सेटरा आपको वाहा मिल जाएगी. इसमें ही शायद आपको आपका ज़्यादा समय डालना होगा| अगर आप अच्छे से यह पढ़े, तोह आपको काफी कुछ पता चल जायेगा Brigade के बारे में|
```

Comment:
"Brigade की कम्पलीट गाइड, कोर कॉम्पोनेन्ट, सिस्टम रोल्स एट्सेटरा आपको वाहा मिल जाएगी." please recheck this. It translates to "**You will get brigade's complete guide**, core components, system roles, etc. there", instead of "**There are complete guides to** Brigade's core components, scripting libraries, system roles and more."

Reply:
"इसमें ही शायद आपको आपका ज़्यादा समय डालना होगा" -> "इसमें ही शायद आप अपना ज़्यादा समय बिताना चाहते हैं।"

Reply:
"vaha mil jayegi" basically points out to विषय गाइड.

Reply:
"अगर आप अच्छे से यह पढ़े, तोह आपको काफी कुछ पता चल जायेगा Brigade के बारे में|" please recheck this too. It conveys "you will know a lot about Bridage" vs "you should come out knowing pretty much everything there is to know about Brigade". We can simply the latter but meaning should not be lost.

Reply:
> "इसमें ही शायद आपको आपका ज़्यादा समय डालना होगा" -> "इसमें ही शायद आप अपना ज़्यादा समय बिताना चाहते हैं।"
Yeah, sure. That would sound better. Thank you!

Reply:
>"vaha mil jayegi" basically points out to विषय गाइड.
Yeah, that part is fine. I was talking about the first part. We are conveying that you will get the complete guide to brigade, core components, etc. as opposed to them getting a **complete guide to** the core components, other things, etc.

Reply:
Can you please suggest me what should I write?

> > "vaha mil jayegi" basically points out to विषय गाइड.
> 
> Yeah, that part is fine. I was talking about the first part. We are conveying that you will get the complete guide to brigade, core components, etc. as opposed to them getting a **complete guide to** the core components, other things, etc.

Reply:
Yeah sure. 
"Brigade की कम्पलीट गाइड, कोर कॉम्पोनेन्ट, सिस्टम रोल्स एट्सेटरा आपको वाहा मिल जाएगी." => "Brigade की कोर कॉम्पोनेन्ट, सिस्टम रोल्स, आदि, की पूर्ण गाइड आपको वाहा मिल जाएगी."